src/Reflection/Pattern.agda

```diff
@@ -98,3 +99,16 @@ absurd ≟ absurd = yes refl
 
 []      ≟s (_ ∷ _) = no λ()
 (_ ∷ _) ≟s []      = no λ()
+
+pattern-size : Pattern → ℕ
+pattern-args-size : List (Arg Pattern) → ℕ
+
+pattern-size (Pattern.con _ ps) = pattern-args-size ps
+pattern-size Pattern.dot = 1
```

Comment:
Minor I know, but any chance we could align these `=` signs?

Reply:
I think I can do that with my space-bar, yes.